packages/api-explorer/src/SecurityInput.jsx

```diff
@@ -37,39 +37,40 @@ class Input extends React.Component {
 }
 
 function SecurityInput(props) {
+  const {auth, scheme} = props
   function change(value) {
-    return props.onChange({ [props.scheme._key]: value });
+    return props.onChange({ [scheme._key]: value });
   }
 
-  switch (props.scheme.type) {
+  switch (scheme.type) {
     case 'apiKey':
     case 'oauth2': {
-      const Component = types[props.scheme.type];
+      const Component = types[scheme.type];
       return (
         <Component
           {...props}
-          apiKey={props.auth[props.scheme._key]}
+          apiKey={auth[scheme._key]}
           change={change}
           Input={Input}
         />
       );
     }
     case 'http':
       // TODO support other schemes? https://github.com/readmeio/api-explorer/issues/15
```

Comment:
Should we remove this TODO?

Reply:
nah.. but ok, I've removed it... It's been there for three years now.. :sweat_smile: